metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -736,6 +748,37 @@ def standardize_schema_table_names(
             raise ValueError(f"schema {schema} does not match table {entity}")
         return segments[0], segments[1]
 
+    def gen_schema_key(self, db_name: str, schema: str) -> PlatformKey:
+        return BigQuerySchemaKey(
+            project_id=db_name,
+            schema=schema,
+            platform=self.platform,
+            instance=self.config.env,
+        )
+
+    def gen_database_key(self, database: str) -> PlatformKey:
```

Comment:
is this a "project id" container? Do you think this is worth having? Perhaps?

Reply:
(vs not having and having a data platform instance later for the project id)

Reply:
Project == Database in bigquery world